bin/Sherpa/cards/production/13TeV/higgs/Run.dat_13TeV_gamgam_forint_interference

```diff
@@ -0,0 +1,45 @@
+ (run){
+ EVENTS 100000;
+ EVENT_GENERATION_MODE P;
+ ME_SIGNAL_GENERATOR Amegic Higgs;
+ RANDOM_SEED XX; XX:=0; WH:=0.00407;
+ QSF:=1.0; SCALES METS{MU_F2}{MU_R2}{QSF*MU_Q2}; CORE_SCALE VAR{Abs2(p[2]+p[3])};
+ HEPMC_USE_NAMED_WEIGHTS=1;
+ SCALE_VARIATIONS 4.0,4.0 1.0,1.0 0.25,0.25 4.0,1.0 1.0,4.0 0.25,1.0 1.0,0.25;
```

Comment:
One last comment, is SCALE_VARIATIONS 1,1 needed? This would be same with your nominal one.

Reply:
I am not sure, I left it there because I found it in all other Sherpa run cards in this repository (e.g. [DY](https://github.com/cms-sw/genproductions/blob/master/bin/Sherpa/cards/production/13TeV/DY_ew/Run.dat_DYee#L8), [exo GJets](https://github.com/cms-sw/genproductions/blob/master/bin/Sherpa/cards/production/13TeV/exo/GJets_Pt-100To200_13TeV-sherpa_cff.py#L30), [VBS](https://github.com/cms-sw/genproductions/blob/master/bin/Sherpa/cards/production/2017/13TeV/VBS/Run.dat_ZA#L8)...).

Reply:
I believe this would be redundant but still fine to be in there.
I think the card is fine @agrohsje @GurpreetSinghChahal